src/bin/fd_filestat_set.rs

```diff
@@ -0,0 +1,107 @@
+use libc;
+use misc_tests::open_scratch_directory;
+use misc_tests::utils::{cleanup_file, close_fd};
+use misc_tests::wasi::{wasi_fd_filestat_set_size, wasi_fd_filestat_set_times, wasi_fd_filestat_get, wasi_path_open};
+use std::{env, process};
+
+fn test_file_allocate(dir_fd: libc::__wasi_fd_t) {
```

Comment:
Would you mind renaming the test case to for example `test_fd_filestat_set`? I know it's less than ideal and somewhat cumbersome, but I'd like to keep the tests names collision free so that, in the future, if anybody wants to, they can pull all test cases into one binary. :-)

Reply:
Sorry for that, that was remains of the old testcase :)